onnx/defs/math/utils.cc

```diff
@@ -69,4 +70,115 @@ from the back. Accepted range is [-r, r-1] where r = rank(input).
     });
   };
 }
+
+void matmulShapeInference(ONNX_NAMESPACE::InferenceContext& ctx, int input1Idx, int input2Idx) {
```

Comment:
```suggestion
void MatmulShapeInference(ONNX_NAMESPACE::InferenceContext& ctx, int input1Idx, int input2Idx) {
```

nit: PascalCase for functions?

Reply:
I haven’t spent long enough time in the C++ codebase but I would suggest https://google.github.io/styleguide/cppguide.html#Function_Names being generally a good guide to follow. More importantly I would just strive for local consistency.

Reply:
Correct. The existing code is a mix of various conventions. I have been recently trying to stick to the Google styleguide, which is what onnxruntime also uses. We should document this somewhere.